src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -11,6 +13,110 @@ import {
 } from '../../../data/slice';
 import messages from './messages';
 
+const data = {
+  'edx-next': {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+  'ed-discuss': {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+  legacy: {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+  piazza: {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+  yellowdig: {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+  inscribe: {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+  discourse: {
+    learnMore: '',
+    configurationDocumentation: '',
+    documentation: '',
+    accessibilityDocumentation: '',
+    emailId: '',
+  },
+};
+
```

Comment:
@awaisdar001 need your opinion on where to place this static data object it will contain all the documentation links once the sheet will be updated